mmv1/third_party/terraform/services/healthcare/resource_healthcare_fhir_store_test.go.erb

```diff
@@ -132,6 +132,7 @@ resource "google_healthcare_fhir_store" "default" {
   disable_referential_integrity = false
   disable_resource_versioning   = false
   enable_history_import         = false
+  enable_history_modifications  = false
```

Comment:
Can you please add the version guard to the new field? 
Example: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/compute/resource_compute_autoscaler_test.go.erb#L305

```
<% unless version == 'ga' -%>

<% end -%>
```

Reply:
Thanks for providing the example! Done.